maploc/data/dataset.py

```diff
@@ -126,44 +127,74 @@ def get_view(self, idx, scene, seq, name, seed, bbox_tile):
+        ).float()
+        world_T_cam2d = Transform2D.from_Transform3D(
+            world_T_cam
+        )  # angle is b/w world x and cam z
+
+        # gcam: gravity-aligned camera with z=optical axis
+        # gcamxyz: gcam rotated such that z:up,x:right,y:forward.
+
+        gcam_angle = world_T_cam2d.angle - 90
+        Rz = Transform2D.from_degrees(gcam_angle, torch.zeros(2))
+        world_R_gcamxyz = torch.eye(3)
+        world_R_gcamxyz[:2, :2] = Rz.R
+        world_T_gcamxyz = Transform3D.from_Rt(world_R_gcamxyz, world_T_cam.t).float()
+        gcamxyz_T_gcam = Transform3D.from_Rt(
+            Rotation.from_euler("X", -90, degrees=True).as_matrix(), torch.zeros(3)
+        ).float()
+        world_T_gcam = world_T_gcamxyz @ gcamxyz_T_gcam
+        gcam_T_cam = world_T_gcam.inv() @ world_T_cam
+
+        world_T_tile = Transform2D.from_Rt(torch.eye(2), canvas.bbox.min_).float()
+        tile_T_cam = world_T_tile.inv() @ world_T_cam2d
+        map_T_cam = Transform2D.from_Rt(
+            tile_T_cam.R, tile_T_cam.t * canvas.ppm - 0.5
+        )  # This will be deprecated, tile_T_cam is sufficient.
+
+        world_T_init = Transform2D.from_Rt(torch.eye(2), bbox_tile.center).float()
+        tile_T_init = world_T_tile.inv() @ world_T_init
+        map_T_init = Transform2D.from_Rt(torch.eye(2), tile_T_init.t * canvas.ppm - 0.5)
+
+        yaw = tile_T_cam.angle
+
         # Map augmentations
-        heading = np.deg2rad(90 - yaw)  # fixme
-        if self.stage == "train":
-            if self.cfg.augmentation.rot90:
-                raster, uv_gt, heading = random_rot90(raster, uv_gt, heading, seed)
-            if self.cfg.augmentation.flip:
-                image, raster, uv_gt, heading = random_flip(
-                    image, raster, uv_gt, heading, seed
-                )
-        yaw = 90 - np.rad2deg(heading)  # fixme
-
-        image, valid, cam, roll, pitch = self.process_image(
-            image, cam, roll, pitch, seed
-        )
+        # TODO: refactor
+        # heading = np.deg2rad(90 - yaw)  # fixme
+        # if self.stage == "train":
+        #     if self.cfg.augmentation.rot90:
+        #         raster, uv_gt, heading = random_rot90(raster, uv_gt, heading, seed)
+        #     if self.cfg.augmentation.flip:
+        #         image, raster, uv_gt, heading = random_flip(
+        #             image, raster, uv_gt, heading, seed
+        #         )
+        # yaw = 90 - np.rad2deg(heading)  # fixme
+
+        cam_R_gcam = gcam_T_cam.inv().R
+        image, valid, cam = self.process_image(image, cam, seed, cam_R_gcam)
+
+        if "plane_params" in self.data:
+            # transform the plane parameters from world to camera frames
+            plane_w = self.data["plane_params"][idx]
+            data["ground_plane"] = torch.cat(
+                [
+                    Transform2D.from_degrees(90 - yaw).R @ plane_w[:2],
+                    plane_w[2:],
+                ]
+            )
```

Comment:
I think that we can remove support for the ground plane, it's not used anywhere anymore.